# This example demonstrates the use of infinite retries for running
# the container to completion. It uses the `random-fail` container.
# For more details, see
# https://github.com/shrinandj/random-fail

apiVersion: argoproj.io/v1alpha1
kind: Workflow
metadata:
  generateName: retry-to-completion-
spec:
  entrypoint: retry-to-completion
  templates:
  - name: retry-to-completion
    retryStrategy: {}
    container:
      image: shrinand/random-fail
      command: ["python"]
      args: ["/run.py", "40"]

